pallets/pallet-dip-consumer/src/lib.rs

```diff
@@ -43,8 +43,9 @@ pub mod pallet {
 	use super::*;
 
 	use frame_support::{
-		dispatch::{Dispatchable, GetDispatchInfo, PostDispatchInfo},
+		dispatch::{GetDispatchInfo, PostDispatchInfo},
 		pallet_prelude::*,
+		sp_runtime::traits::Dispatchable,
```

Comment:
`sp_runtime` directly.

Reply:
The pallet does not have `sp_runtime`. Should I add it?

Reply:
As above.